source/12_policies/edac.rst

```diff
@@ -0,0 +1,156 @@
+Uncertainty-Based Offline Reinforcement Learning with Diversified Q-Ensemble (EDAC), first proposed in `Uncertainty-Based Offline Reinforcement Learning with Diversified Q-Ensemble <https://arxiv.org/pdf/2110.01548.pdf>`_, 
+is one of them which penalty out-of-distribution(OOD) action by add critic. 
+
+Quick Facts
+-------------
+1. EDAC is an offline RL algorithm.
+
+2. EDAC can be implemented with less than 20 lines of code on top of a
+   number of SAC RL algorithms
+
+3. EDAC supports both **discrete** and **continuous** action spaces.
+
+
+Key Equations or Key Graphs
+---------------------------
+EDAC show that cliped Q-learning :math:`min_{j=1,2}Q(s,a)` can penalty OOD action by add the number of critic. 
+Therefore EDAC can be implemented by ensembleing Q-network on standard SAC RL algorithm and add a penalty term to penalty OOD action.
+
+In general, for the EDAC, the penalty term is following:
+
+.. image:: images/edac_penalty_term.png
+   :align: center
+   :scale: 55%
+
+By adding above penalty term, the algorithm can minimize the number of necessary critics to achieve higher computational efficiency while maintaining good performance. This term compute the inner-product of gradients of Q-values of in-dataset state-actions.
+
+EDAC show the importance of cliped Q-learning by increasing the number of Q-networks in SAC algorithm. By computeing the clipe penalty and standard deviation
+of OOD action adn in-dataset action, paper show why algorithm will perform better by increasing the number of Q-network. The figure of these results is as following:
+
+.. image:: images/edac_clip_penalty.png
+   :align: center
+   :scale: 55%
+
+In the figure below, paper shows the gradients of Q-value of OOD action, where the red vector indicates the direction under OOD actions. We find that if the 
+variance of Q-value of OOD actions is small, then it will receive a smaller penalty during gradient descent. This further explains why increasing the number of
+critics can penalize OOD actions, and also indicates the direction for later improvement in the paper.
+
+.. image:: images/edac_Q_grad.png
+   :align: center
+   :scale: 55%
+
+Althought increasing the number of Q-networks can improve performance, too many Q networks will become a burden. Therefore, EDAC add a penalty term to reduce the number of critics.
+Using first-order Taylor approximate, he sample variance of the Q-values at an OOD action along wcan be represented as following:
+
+.. image:: images/edac_taylor.png
+   :align: center
+   :scale: 55%
+
+To increasing the variance of Q-values effectively for near-distribution OOD actions, EDAC maximize the following equation:
+
+.. image:: images/edac_maxize.png
+   :align: center
+   :scale: 55%
+
+There are several methods to compute the smallest eigenvalue, such as the power method or the QR algorithm [ 27]. However, these iterative methods require 
```

Comment:
you can omit this paragraph and delete lemma1, if you just copy the raw sentence of the paper